src/components/Receiving/ReceivingList.js

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import {
+  get,
+  uniqBy,
+} from 'lodash';
+
+import {
+  Button,
+  IconButton,
+  MultiColumnList,
+  Pane,
+  PaneMenu,
+  Paneset,
+  Row,
+  SearchField,
+  TextField,
+} from '@folio/stripes/components';
+
+import { RECEIVING_API } from '../Utils/api';
+import FolioFormattedTime from '../FolioFormattedTime';
+import css from './ReceivingList.css';
+
+class ReceivingList extends Component {
+  static manifest = Object.freeze({
+    receiving_history: {
+      type: 'okapi',
+      path: RECEIVING_API,
+      throwErrors: false,
+    },
+  })
+
+  static propTypes = {
+    resources: PropTypes.object,
+    location: PropTypes.object.isRequired,
+    history: PropTypes.shape({
+      replace: PropTypes.object.isRequired,
+    }).isRequired,
+  }
+
+  onCloseReceiving = () => {
+    const { history, location } = this.props;
+
+    history.replace(location.pathname.replace('/receiving', ''));
+  }
+
+  getFirstMenu = () => (
+    <PaneMenu>
+      <FormattedMessage id="ui-orders.buttons.line.close">
+        {(title) => (
+          <IconButton
+            aria-label={<FormattedMessage id="ui-orders.buttons.line.closeDialog" />}
+            icon="times"
+            id="clickable-close-new-line-dialog"
+            onClick={this.onCloseReceiving}
+            title={title}
+          />
+        )}
+      </FormattedMessage>
+    </PaneMenu>
+  );
+
+  render() {
+    const { resources } = this.props;
+    const receiving = get(get(resources, ['receiving_history', 'records', 0], []));
```

Comment:
records[0] could be an array? I'm not sure. But as the default value you've put an array. It looks like no consistency to me

Reply:
Also the name of variable is too general